tests/maas-install.sh

```diff
@@ -18,10 +18,29 @@
 
 set -evu
 
+# sets the desired maas_release to test
+MAAS_RELEASE=master
+
+# Ensure we have one copy of the maas_release variable since
```

Comment:
maybe this would just be easier if we passed the needed variable into the required playbooks? 

``` shell
openstack-anisble maas-get.yml -e maas_release="${MAAS_RELEASE}"
```

Reply:
Yeah... that probably would be easier... I'll make the change to use that instead.